app/assets/stylesheets/trace_view/_right_menu.less

```diff
@@ -21,6 +21,18 @@
   }
 }
 
+.info-tab-content {
+  padding: 10px;
+}
+
+.lm_header {
+  height: 20px !important;
+}
+
+.lm_splitter.lm_vertical {
+  padding-bottom: 2px;
```

Comment:
I would hope that you don't need to set this if you follow my previous comment :)